cmd/sidecar/main.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/cenkalti/backoff"
 	"github.com/cyberark/conjur-authn-k8s-client/pkg/authenticator"
 	authnConfig "github.com/cyberark/conjur-authn-k8s-client/pkg/authenticator/config"
-	"github.com/cyberark/conjur-authn-k8s-client/pkg/storage"
+	storageHandler "github.com/cyberark/conjur-authn-k8s-client/pkg/storage"
```

Comment:
i think this should be `storage` (so you can just remove the `storageHandler`). this object creates a hadnler